content/md/imports-exports/product-export-builder.md

```diff
@@ -143,21 +143,24 @@ The following drop down enables you to filter on the completeness of the product
 
 ### Filter on date
 
-You can now export your product on a specific time condition:
-- `No date condition` (default option)
-- `Updated products over the last n days (e.g. 6)`: you want to export products updated since the last ‘n’ days:
-- `Updated products since this date`:  you want to export products updated since a specific date
-- `Updated products since last export`: you want to export products updated since the last export
+You can now export your products based on a specific time condition:
+- `No date condition` (default option): no time condition will not be taken into account for your product export
+- `Updated products over the last n days (e.g. 6)`: you only want to export the products that have been updated since the last ‘n’ days
+- `Updated products since this date`: you only want to export the products that have been updated since a specific date
+- `Updated products since last export`: you only want to export the products that have been updated since the last export
 
 If you select `Updated products over the last n days (e.g. 6)`, a field will be displayed nearby the drop down menu. Specify a number of days (please only use numbers) in this field.
 
-If you select `Updated products since this date`, the date picker will be displayed to choose a date. To change the date, click again on the date picker. To completely remove a date, select another time condition.
+If you select `Updated products since this date`, click the empty field to display the date picker and choose a date. To change the date, click again on the date picker. To completely remove a date, select another time condition.
+
+::: tips
+Save export time and used this filter to only export edited data:::
```

Comment:
I have a problem of display with the tips, the filter on categories is added in the tips.
![slack_-_akeneo](https://user-images.githubusercontent.com/10232850/40984385-126632a2-68e2-11e8-860d-7358ab381bdc.png)

Reply:
Like this?
::: tips
Save export time and used this filter to only export edited data
:::

Reply:
Yup !